### PR TITLE
Improve the navbar #2

### PR DESCRIPTION
**Pull request by ZoranPandovski**
#2

### CODE REVIEW

**Comment by ZoranPandovski** [comment]
#2

File: home/templates/home.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {% load staticfiles %}
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+<head>
```
Comment (divyam3897) [inline home/templates/home.html]: Remove this change

File: home/templates/home.html [hunk 2]
```diff
@@ -10,28 +10,41 @@
...
+    <link rel="stylesheet"
+          href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css"
+          integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp"
+          crossorigin="anonymous">
+    <link rel="stylesheet" href="./static/base.css">
+    <!-- Add Jquery -->
+    <script
+            src="https://code.jquery.com/jquery-3.2.1.min.js"
+            integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4="
+            crossorigin="anonymous"></script>
```
Comment (divyam3897) [inline home/templates/home.html]: Add only the extra lines and remove the non required changes

File: home/templates/home.html [hunk 3]
```diff
@@ -10,28 +10,41 @@
...
-  </head>
-  <body  style="background-image: url({% static 'images/background.jpg' %});background-size:cover;">
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"
+            integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa"
+            crossorigin="anonymous"></script>
+<style>
 
+</style>
+</head>
+<body>
```
Comment (divyam3897) [inline home/templates/home.html]: Remove the changes.

File: home/templates/home.html [hunk 4]
```diff
@@ -10,28 +10,41 @@
...
     </div>
-    <ul class="nav navbar-nav">
-      <li class="active"><a href="#">Home</a></li>
-      <li><a href="#">Resources</a></li>
-    </ul>
-  </div>
 </nav>
-
-  </body>
+</body>
```
Comment (divyam3897) [inline home/templates/home.html]: Remove the change

File: home/templates/home.html [hunk 5]
```diff
@@ -10,28 +10,41 @@
...
-    <div class="navbar-header">
-      <a class="navbar-brand" href="#">Programming club</a>
+    <div class="container-fluid">
+        <div class="navbar-header">
+            <a class="navbar-brand" href="#">Programming club</a>
+        </div>
+        <nav class="nav nav-masthead header">
+            <a class="nav-link active" href="#">Home</a>
+            <a class="nav-link" href="#">Resources</a>
+        </nav>
```
Comment (divyam3897) [inline home/templates/home.html]: Keep only the changes done, remove everything else.

File: pclubWebsite/settings.py [hunk 6]
```diff
@@ -120,7 +120,6 @@
 
 STATIC_URL = '/static/'
 STATIC_ROOT = os.path.join(BASE_DIR,'home','static')
-STATICFILES_DIRS = [ os.path.join(BASE_DIR,'home','staticFiles') ]
```
Comment (divyam3897) [inline pclubWebsite/settings.py]: why did you remove this?
